src/flashops.c

```diff
@@ -1,8 +1,8 @@
 #include "flashops.h"
 
+#include <memory.h>
```

Comment:
Not necessary, just use string.h as that is the standard include needed for memset/clr/etc()